Library/.rubocop_rules.yml

```diff
@@ -61,6 +61,14 @@ Lint/AssignmentInCondition:
 Lint/ParenthesesAsGroupedExpression:
   Enabled: false
 
+Lint/UnneededSplatExpansion:
+  Exclude:
+    - 'Taps/**/*'
+
+Style/NumericPredicate:
+  Exclude:
+    - 'Taps/**/*'
```

Comment:
Yes, that's what I thought right *after* i pushed this and saw your PR on `hombrew/core`. =)